.github/workflows/container-tests.yml

```diff
@@ -77,16 +77,39 @@ jobs:
         with:
           ref: "refs/pull/${{ github.event.issue.number }}/head"
 
+      - name: Prepare needed variables
+        shell: bash
+        id: vars
+        run: |
+          dockerfile="Dockerfile.${{ matrix.os_test }}"
+          if [[ ${{ matrix.os_test }} == "centos7" ]]; then
+            dockerfile="Dockerfile"
+          fi
+          echo "::set-output name=DOCKERFILE_NAME::${dockerfile}"
+
+      - name: Check ${{ matrix.version }}/${{ steps.vars.outputs.DOCKERFILE_NAME }} presents
+        id: check_dockerfile
+        uses: andstor/file-existence-action@v1
+        with:
+          files: "${{ matrix.version }}/${{ steps.vars.outputs.DOCKERFILE_NAME }}"
+
+      - name: Check .exclude-${{ matrix.os_test }} file presents
+        id: check_exclude_file
+        uses: andstor/file-existence-action@v1
+        with:
+          files: "${{ matrix.version }}/.exclude-${{ matrix.os_test }}"
+
       # https://github.com/sclorg/testing-farm-as-github-action
-      - name: Schedule tests on external Testing Farm by Testing-Farm-as-github-action
+      - name: Schedule tests for ${{ matrix.context }} - ${{ matrix.version }}
         id: github_action
+        if: ${{ steps.check_exclude_file.outputs.files_exists == 'false' && steps.check_dockerfile.outputs.files_exists == 'true' }}
         uses: sclorg/testing-farm-as-github-action@v1
         with:
           api_key: ${{ secrets[matrix.api_key] }}
           git_url: ${{ matrix.tmt_repo }}
           git_ref: ${{ matrix.branch }}
           tf_scope: ${{ matrix.tf_scope }}
           tmt_plan_regex: ${{ matrix.tmt_plan }}
-          pull_request_status_name: ${{ matrix.context }}
-          variables: "REPO_URL=$GITHUB_SERVER_URL/$GITHUB_REPOSITORY;REPO_NAME=$GITHUB_REPOSITORY;PR_NUMBER=${{ github.event.issue.number }};OS=${{ matrix.os_test }};TEST_NAME=test"
+          pull_request_status_name: "${{ matrix.context }} - ${{ matrix.version }}"
+          variables: "REPO_URL=$GITHUB_SERVER_URL/$GITHUB_REPOSITORY;REPO_NAME=$GITHUB_REPOSITORY;PR_NUMBER=${{ github.event.issue.number }};VERSION=${{ matrix.version }};OS=${{ matrix.os_test }};TEST_NAME=test"
```

Comment:
```suggestion
          variables: "REPO_URL=$GITHUB_SERVER_URL/$GITHUB_REPOSITORY;REPO_NAME=$GITHUB_REPOSITORY;PR_NUMBER=${{ github.event.issue.number }};SINGLE_VERSION=${{ matrix.version }};OS=${{ matrix.os_test }};TEST_NAME=test"
```

Reply:
In TF we use VERSION https://github.com/sclorg/sclorg-testing-farm/blob/main/plans/c8s.fmf#L10
In my opinion, SINGLE_VERSION does not make sense here. But it is my opinion.

Reply:
Right, sorry.

Reply:
Too many VERSIONS :).